helm/charts/determined/templates/master-deployment.yaml

```diff
@@ -40,11 +40,13 @@ spec:
           - name: master-config
             mountPath: /etc/determined/
             readOnly: true
-          {{- if .Values.tlsSecret }}
+          {{- if .Values.security }}
+          {{- if .Values.security.tlsSecret }}
           - name: tls-secret
             mountPath: {{ include "determined.secretPath" . }}
             readOnly: true
           {{ end }}
+          {{ end }}
```

Comment:
We're oddly inconsistent with `{{ end }}`, `{{- end }}`, and `{{ end -}}` i this chart.  Maybe not really something that belongs in this PR, but probably something worth cleaning up.